app/src/main/java/com/example/parkingsystem/mvp/model/ParkingModel.java

```diff
@@ -1,16 +1,11 @@
 package com.example.parkingsystem.mvp.model;
 
-import com.example.parkingsystem.mvp.contract.ParkingContract;
+import com.example.parkingsystem.mvp.contracts.ParkingContract;
 
 public class ParkingModel implements ParkingContract.ParkingModel {
 
-    private int parkingLots;
-
-    public int getParkingLots() {
-        return parkingLots;
-    }
-
-    public void setParkingLots(int parkingLots) {
-        this.parkingLots = parkingLots;
```

Comment:
why you remove the set method? How you update in the model the number value changed?

Reply:
You're right, I'll add the set method again!